docs/contributing/using-instrumenter-api.md

```diff
@@ -116,6 +116,33 @@ The `builder()` method accepts three arguments:
 An `Instrumenter` can be built from several smaller components. The following subsections describe
 all interfaces that can be used to customize an `Instrumenter`.
 
+### Set the instrumentation version and OpenTelemetry schema URL
+
+Make sure you always provide the instrumentation library version to the `Instrumenter`. You can do
+this in two ways:
+
+* By calling the `setInstrumentationVersion()` method on the `InstrumenterBuilder`.
+* By making sure the JAR with your instrumentation library contains a properties file with the
+  version. The file must be located in the `META-INF/io/opentelemetry/instrumentation/` directory,
+  and it must be named `${instrumentationName}.properties`, where `${instrumentationName}` is the
+  name of the instrumentation library that is passed to the `Instrumenter#builder()` method. The
+  file must contain a single property `version`, e.g.:
+
+    ```properties
+    # META-INF/io/opentelemetry/instrumentation/my-instrumentation.properties
+    version = 1.2.3
+    ```
+
+  The `Instrumenter` will automatically detect these properties files and determine the
+  instrumentation version based on its name.
+
+By setting the version, you let users identify which version of your instrumentation produced the
+telemetry.
```

Comment:
Since this explains the importance of doing this, I'd move this paragraph right after the heading, before "Make sure..."